src/Window.cpp

```diff
@@ -237,13 +243,28 @@ bool Gosu::Window::tick()
     }
     
     SDL_Event e;
+
     while (SDL_PollEvent(&e)) {
-        if (e.type == SDL_QUIT) {
-            SDL_HideWindow(shared_window());
-            return false;
-        }
-        else {
-            input().feed_sdl_event(&e);
+        switch (e.type) {
+            case (SDL_QUIT): {
+                SDL_HideWindow(shared_window());
+                return false;
+            }
+        #if SDL_VERSION_ATLEAST(2, 0, 5)
+            case (SDL_DROPBEGIN):
+            case (SDL_DROPCOMPLETE):
+            case (SDL_DROPTEXT):
```

Comment:
As I dont have SDL 2.0.5 I couldn't really test what the other events do / return. How I interpret the config is that they all return the filename but are triggered by different things? So I thought I just add all of the SDL_DropEvent types here.